drivers/SmartThings/philips-hue/src/init.lua

```diff
@@ -191,7 +195,8 @@ bridge_added = function(driver, device)
   local device_bridge_id = device.device_network_id
 
   if not driver.joined_bridges[device_bridge_id] then
-    Discovery.search_for_bridges(driver, function(driver, bridge_ip, bridge_id)
+    local known_macs = {}
+    Discovery.search_for_bridges(driver, known_macs, function(driver, bridge_ip, bridge_id)
```

Comment:
Couldn't you just pass in an explicit `{}`? I must be missing something.

Reply:
Absolutely could, wanted to leave this here to hang my hat off of if I find out later I could actually have done something to optimize this. I am *fatigued*.